pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.9</version>
+        <version>2.22</version>
```

Comment:
I think this broke the build due to https://github.com/jenkinsci/plugin-pom/pull/36. Missed this one during my scan, probably because `ui/` is a subdirectory.